datahub-web-react/src/app/entity/mlFeatureTable/profile/features/TableOfMlFeatures.tsx

```diff
@@ -0,0 +1,155 @@
+        align: 'left' as AlignType,
+        render: (dataType: MlFeatureDataType) => {
+            return <MlFeatureDataTypeIcon dataType={dataType} />;
+        },
+    },
+];
+
+type Props = {
+    features: Array<MlFeature | MlPrimaryKey>;
+};
+
+export default function TableOfMlFeatures({ features }: Props) {
+    const refetch = useRefetch();
+    const [updateDescription] = useUpdateDescriptionMutation();
+    const entityRegistry = useEntityRegistry();
+
+    const [tagHoveredIndex, setTagHoveredIndex] = useState<string | undefined>(undefined);
+
+    const onTagTermCell = (record: any, rowIndex: number | undefined) => ({
+        onMouseEnter: () => {
+            setTagHoveredIndex(`${record.urn}-${rowIndex}`);
+        },
+        onMouseLeave: () => {
+            setTagHoveredIndex(undefined);
+        },
+    });
+
+    const nameColumn = {
+        title: 'Name',
+        dataIndex: 'name',
+        key: 'name',
+        width: 100,
+        render: (name: string, feature: MlFeature | MlPrimaryKey) => (
+            <Link to={entityRegistry.getEntityUrl(feature.type, feature.urn)}>
+                <Typography.Text strong>{name}</Typography.Text>
+            </Link>
+        ),
+    };
+
+    const descriptionColumn = {
+        title: 'Description',
+        dataIndex: 'description',
+        key: 'description',
+        render: (_, feature: MlFeature | MlPrimaryKey) => (
+            <SchemaDescriptionField
+                description={feature?.editableProperties?.description || feature?.properties?.description || ''}
+                original={feature?.properties?.description}
+                isEdited={!!feature?.editableProperties?.description}
+                onUpdate={(updatedDescription) =>
+                    updateDescription({
+                        variables: {
+                            input: {
+                                description: updatedDescription,
+                                resourceUrn: feature.urn,
+                            },
+                        },
+                    }).then(refetch)
+                }
+            />
+        ),
+        width: 300,
+    };
+
+    const tagColumn = {
+        width: 125,
+        title: 'Tags',
+        dataIndex: 'tags',
+        key: 'tags',
+        render: (_, feature: MlFeature | MlPrimaryKey, rowIndex: number) => (
+            <TagTermGroup
+                editableTags={feature.tags}
+                canRemove
+                buttonProps={{ size: 'small' }}
+                canAddTag={tagHoveredIndex === `${feature.urn}-${rowIndex}`}
+                onOpenModal={() => setTagHoveredIndex(undefined)}
+                entityUrn={feature.urn}
+                entityType={feature.type}
+                refetch={refetch}
+            />
+        ),
+        onCell: onTagTermCell,
+    };
+
+    const termColumn = {
+        width: 125,
+        title: 'Terms',
+        dataIndex: 'glossaryTerms',
+        key: 'glossarTerms',
```

Comment:
is this misnamed?

Reply:
Ah I see it's like that in the original as well. Can we change it?

Reply:
done!